services/web/client/source/class/qxapp/theme/Decoration.js

```diff
@@ -18,5 +18,22 @@
 qx.Theme.define("qxapp.theme.Decoration", {
   extend: osparc.theme.osparcdark.Decoration,
 
-  decorations: {}
+  decorations: {
+    "droppableWidget": {
+      decorator: qx.ui.decoration.MSingleBorder,
```

Comment:
I would use `qx.ui.decoration.Decorator`  as it includes all the decoration mixins